homework/homework9/find_specified_character.py

```diff
@@ -0,0 +1,39 @@
+"""Find specific character.
+This program finds the character “#”.
+If this character is found - removes the previous character from the string.
+"""
+
+
+def find_specific_character(data: str) -> str:
+    """
+    This function finds the “#” character and
+    if this character is found - deletes the previous character
+    from the string
+
+    data: str
```

Comment:
pls, add more information here. 
https://google.github.io/styleguide/pyguide.html#383-functions-and-methods